dev/MRTCore/mrt/Core/src/MRM.cpp

```diff
@@ -677,7 +677,7 @@ STDAPI MrmGetChildResourceMap(
     }
 
     const ResourceMapSubtree* childSubTree;
-    RETURN_IF_FAILED(originalMapSubtree->GetSubtree(childResourceMapName, &childSubTree));
+    RETURN_IF_FAILED_WITH_EXPECTED(originalMapSubtree->GetSubtree(childResourceMapName, &childSubTree), HRESULT_FROM_WIN32(ERROR_MRM_MAP_NOT_FOUND));
```

Comment:
>RETURN_IF_FAILED_WITH_EXPECTED [](http://example.com/codeflow?start=4&length=30)

What's the difference between RETURN_IF_FAILED and RETURN_IF_FAILED_WITH_EXPECTED? Spent some time trying to understand from the definitions. It's non-obvious...